tool/tctl/common/resource_command.go

```diff
@@ -37,6 +37,7 @@ import (
 	log "github.com/sirupsen/logrus"
 	"google.golang.org/protobuf/encoding/protojson"
 	kyaml "k8s.io/apimachinery/pkg/util/yaml"
+	"k8s.io/kubectl/pkg/util/term"
```

Comment:
I tried using the native `"golang.org/x/term"` but it had different behavior from k8s package. Golang `term` return false.

Reply:
Ok. I was just using the API wrong. Moved to proper package